src/components/hero/styles.ts

```diff
@@ -0,0 +1,48 @@
+import theme from '@mdb/flora/theme';
+
+export const breadcrumbsContainerStyles = {
+    alignItems: 'center',
+    display: 'flex',
+    flexWrap: 'wrap' as 'wrap',
+    gap: 'elementXSmall',
+    gridColumn: ['span 6', null, 'span 8', 'span 12'],
+};
+
+export const breadcrumbStyles = {
+    alignItems: 'center',
+    display: 'flex',
+    gap: 'elementXSmall',
+};
+
+export const linkStyles = {
+    'div > span.textlink-default-text-class': {
```

Comment:
I would be careful with targeting the HTML tag here since if the tag changes these CSS rules wouldn't apply anymore. If there is already a class on it I would target that alone.